### DISCUSSION TITLE
How to configure?

### DISCUSSION THREAD

**Discussion by Twooearly** [RaspAP/raspap-webgui#1535] [Q&A] [answered]
So, I got rasp ap up and running and can access it through the connected LAN. I use a raspberry pi 0 w, with a micro usb Ethernet adapter. Maybe not the perfect setup but I wanted exactly this formfactor. I connected the pi through the Ethernet adapter to my home network and it also shows up and I can ssh into it or can access the webui. Now I want to take the connected Ethernet as the input and output the internet through the hotspot. So I can connect my mobile devices. I want the raspi to create it's own network with it's own dhcp and pi addresses. So my question: how do I have to configure the webui to achieve this? Thanks in advance!

**Comment by billz** [comment, marked as answer]
> I want to take the connected Ethernet as the input and output the internet through the hotspot

What you describe is RaspAP's most basic and [default configuration](https://docs.raspap.com/defaults/). ie., internet supplied by `eth0` and a hotspot created with the onboard wireless adapter. If you haven't modified settings in the UI, simply start the hotspot. 

If you have modified settings, do a System reset (explained in the docs).
